lib/Auth/Process/ProxyFilter.php

```diff
@@ -9,13 +9,14 @@
 /**
  * Class sspmod_perun_Auth_Process_ProxyFilter
  *
- * This filter allows to disable nested filter for particular SP
- * or for users with one of (black)listed attribute values.
- * When any of the values matches, the nested filter is NOT run.
+ * This filter allows to disable/enable nested filters for particular SP
+ * or for users with one of (black/white)listed attribute values.
+ * When any of the values matches, the nested filters do NOT run (blacklist) or run (whitelist).
```

Comment:
This sentence does not make sense. If I understand it correctly, based on the mode of operation of the filter or whitelist mode, the filters are or are not run. If this is the case, I would restructure it to something like:

> Based on the mode of operation, the nested filters ARE (whitelist) or ARE NOT (blacklist) run when any of the attribute values match.